docs/platforms/apple/guides/ios/session-replay/performance-overhead.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Performance Overhead
+sidebar_order: 5502
+notSupported:
+description: "Learn more about how enabling Session Replay impacts the performance of your application."
+---
+
+The only way to get accurate metrics is to measure performance overhead yourself. You can apply realistic access patterns against your own application, and correlate that to your own topline business metrics. You can learn more about the various optimizations implemented in the iOS Replay SDK in the [Replay Performance Overhead](/product/explore/session-replay/mobile/performance-overhead/) documentation.
+
+We measured the overhead of the iOS Replay SDK using the [Pocket Casts](https://github.com/Automattic/pocket-casts-ios) open-source app, which includes a diverse set of components such as ViewControllers and SwiftUI screens. The benchmarks were conducted with full masking enabled, using optimized release builds, and running through the same user flow 10 times. This approach ensured the measurements closely reflect the overhead in a real-world application. Here are the results (median values are shown):
+
+| Metric                           | Sentry SDK only | Sentry + Replay SDK |
+| -------------------------------- | --------------- | ------------------- |
+| FPS                              | 55 fps          | 53 fps              |
+| Memory                           | 102 MB          | 121 MB              |
+| CPU                              | 4%              | 13%                 |
+| App Startup Time (Cold)          | 1264.80 ms      | 1265 ms             |
+| Main Thread Time                 | -               | 43ms                |
```

Comment:
why do "Main Thread Time" and "Network Bandwidth" have "-' out of curiosity? I would do "n/a" is they're not applicable :)

Reply:
yeah I can change it to "n/a". Certainly there's some overhead on the main thread and network bandwidth without the replay part, but it's not like replay is adding "on top" of that, it's kind of completely independent and happens at a different point in time.